eahub/profiles/views.py

```diff
@@ -178,18 +179,37 @@ def edit_profile_community(request):
     if request.method == "POST":
         form = EditProfileCommunityForm(request.POST, instance=request.user.profile)
         if form.is_valid():
+            old_local_groups = [
+                x.name
+                for x in Profile.objects.filter(id=request.user.profile.id)[
+                    0
+                ].local_groups.all()
+            ]
```

Comment:
Can we do something as `[group.name for group in LocalGroup.objects.filter(membership__profile= request.user.profile)]?`

I would also recommend to do it on the line №208 to follow the variable lifespan guidelines (the variable is be defined as close to its usage as possible to minimize the cognitive load of the programmer)